packages/icons/src/ui.js

```diff
@@ -303,7 +303,7 @@ export {
   Time,
   TimeFilled,
   Token,
-  Tolietries,
```

Comment:
Is there a chance this change will break some application that is using this icon with the wrong name?

Reply:
good point that's why we are trying to address it as a breaking change

Reply:
what do you think?

Reply:
I like the approach of addressing this as a breaking change. In addition, I would also suggest to keep two exports: one with the wrong name (just like it is right now), and another one with your fix.

```jsx
export {
...
Tolietries,
Toiletries: Tolietries,
....
}
```

That way, we allow teams time to fix the problem before it happens. That way, we allow teams time to fix the problem before it happens. And therefore, we have set a deadline for the incorrect version to be removed, such as 30 days. WDYT?

CC @matheushdsbr @caioalexandrebr